### ISSUE TITLE
wrong conv,  no spare, no likehood train

### ISSUE THREAD

**Issue by ddz16** [mlpotter/Transformer_Time_Series#3]
Thanks for your code. But I think your code is very different from the method which the paper mentioned.

**Comment by ChenS676** [comment]
I just found in the implementation the output is used in combination with input, where is obviously not the real case in practise. If you just use input performance decays about 10%.

**Comment by WLW2980** [comment]
yes , I agree with @ChenS676. The validate part is wrong, beacuse the truth target is also input and it will cause 'cheating' since In fact we do not know the true value(target here) when using model to predict
